pyracf/access/access_admin.py

```diff
@@ -40,13 +40,15 @@ def __init__(
         generate_requests_only: bool = False,
         add_field_data: Union[dict, None] = None,
         overwrite_field_data: Union[dict, None] = None,
+        add_more_secrets: Union[dict, None] = None,
```

Comment:
I would call this field `additional_secret_traits`.

Reply:
agree; that's better. change made